README.md

```diff
@@ -1,105 +1,64 @@
-
 ---
+# Whats Cookin'?
+###### Meals made easy!
+---
+## Table of Contents
+* [Introduction](#introduction)
+* [Features](#features)
+* [Technologies](#technologies)
+* [Deployment](#deployment)
+* [Authors](#authors)
 
-## Test Files Organization
-
-Similar to feature code, your test code needs to be put in a specific place for it to run successfully.
-
-**Put all of your test files in the `test` directory.** As a convention, all test filenames should end with `-test.js`. For instance: `box-test.js`.
-
-## Running Your Tests
+## Introduction
+The primary goal of [Whats-Cookin](https://github.com/alia-peterson/whats-cooking) is to help users plan their meals. They can search through a log of recipes, save their favorite recipes, and keep track of their pantry. The specifications can be found [here](https://frontend.turing.io/projects/whats-cookin.html) and [here](https://frontend.turing.io/projects/module-2/refactor-tractor-wc.html)
 
-Run your test suite using the command:
+## Motivation
+The motivation behind this project's creation was to get experience working in an existing code base. It also allowed the opportunity to ractice using `Fetch` API and updating code to incorporate more modern functionality. 
 
-```bash
-npm test
-```
+---
 
-The test results will output to the terminal.
+## Features
+* [Search Ideas](#Search-Ideas)
 
----
 
-## Linting Your Code
 
-Run the command in your terminal `npm run lint` to run the linter on your JavaScript code. There will be errors and warnings right from the start in this starter kit - the linter is still running successfully.
 
-Your linter will look at the JavaScript files you have within the `src` directory and the `test` directory. 
+#### Search Ideas
+You'll find all of the recipes you need here, but that can be a lot to handle, so we got a search bar, baby! Just type in a recipe name or ingredient, and we'll find everything that suits your needs.  
+<p align = "center">
+<img src="https://lwgsummerland.files.wordpress.com/2013/06/sexyvegeman.jpg">
+</p>
+    <details>
+      <summary>Under the Hood</summary>
+      Some Stuff
+    </details>
 
-## Webpack?
 
-If you look in the `package.json` file, you'll see one of the library dependencies called `webpack`. If you're interested in learning more about what Webpack is and how it works behind the scenes, take a look through the [Webpack configuration documentation](https://webpack.js.org/concepts/).
 
-## Deploying to GitHub Pages
+#### Continuous Improvement/Future Improvements
 
-_If you are finished with the functionality and testing of your project_, then you can consider deploying your project to the web! This way anyone can play it without cloning down your repo.
+---
 
-[GitHub Pages](https://pages.github.com/) is a great way to deploy your project to the web. Don't worry about this until your project is free of bugs and well tested!
+## Technologies
+JS, Fetch/Async, HTML, & CSS/SCSS, Normalize, Webpack
```

Comment:
- Remove &
- Could maybe include JSON, Mocha, and Chai?

Reply:
good call